src/InfluxDB/Point.php

```diff
@@ -81,26 +88,39 @@ public function __toString()
         return $string;
     }
 
+    /**
+     * @param  array $arr
+     * @return string
+     */
     private function arrayToString(array $arr)
     {
         $strParts = array();
 
         foreach ($arr as $key => $value) {
-            $strParts[] = "{$key}={$value}";
+            $strParts[] = sprintf('%s=%s', $key, $value);
```

Comment:
It also makes it more readable. And I'd say that readability in this case is better than `0.000000000001` nanoseconds of performance difference.